src/components/SecondaryPanes/Expressions.js

```diff
@@ -256,8 +254,7 @@ class Expressions extends Component<Props, State> {
 
   renderAutoCompleteMatches() {
     const { autocompleteMatches } = this.props;
-    const { enableAutocomplete } = this.state;
-    if (autocompleteMatches && enableAutocomplete) {
+    if (autocompleteMatches && features.autocompleteExpression) {
```

Comment:
`(features.autocompleteExpression && autocompleteMatches)` would be slight better just so it fails immediately when it isn't enabled.